frontend/packages/process-editor/src/components/ConfigPanel/EditDataTypesToSign/SelectDataTypesToSign/SelectDataTypesToSign.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { Combobox } from '@digdir/design-system-react';
+import { StudioButton } from '@studio/components';
+import { useTranslation } from 'react-i18next';
+import { XMarkIcon } from '@studio/icons';
+import classes from './SelectDataTypesToSign.module.css';
+import { useBpmnContext } from '@altinn/process-editor/contexts/BpmnContext';
```

Comment:
Until we have decided 100% how to do imports, we should aim to use relative imports to files in the same package 😄 
Same for line 12

Reply:
Done!